bitvision/engine/data_bus.py

```diff
@@ -11,6 +11,10 @@
 import requests
 from bs4 import BeautifulSoup
 from ftfy import ftfy
+
+# This shouldn't have to be here but I can't get it to work without it. idfk
+import sys
+sys.path.insert(0, "engine")
```

Comment:
Just add a dot before `transformers`, like so: `from .transformers import Transformer`

Reply:
Why does that work?

Reply:
`.` represents the dir you're in (i.e. `engine/`), just like `..` represents the parent dir.

Reply:
Shouldn't that be implied?